src/utils/bitmap.rs

```diff
@@ -67,13 +67,38 @@ fn load_from_png_file(filename: &str) -> Bitmap {
     };
 }
 
+/// # Panics
```

Comment:
I suppose we need custom error here, so we can signal it to the cli user correctly

Reply:
Also I suppose we need some sort of tooling to make us mark such kind of docs